cmd/charm/charmcmd/cmd.go

```diff
@@ -55,6 +55,7 @@ func New() *cmd.SuperCommand {
 			DefaultConfig: os.Getenv(osenv.JujuLoggingConfigEnvKey),
 		},
 		NotifyHelp: notifyHelp,
+		Version:    "2.1.1",
```

Comment:
I can see this getting out of date very quickly. Would it be possible to use the Makefile to set this based on tags like in charmstore?